blocks/shared-styles/_children/headings/section/index.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+import LevelContext from '../context';
+
+const HeadingSection = (props) => (
+  <LevelContext.Consumer>
+    {(level) => (
+      <LevelContext.Provider value={level + 1}>
```

Comment:
cool. I guess we're accounting for h7+ issues with the Math.min. Could we have a custom setter here that maxes out at 6? suggestion, not necessarily criticism

Reply:
That's in the Heading it will max out at 6